### PR TITLE
vim: add new examples for opening vim and refer to vimtutor

### PR DESCRIPTION
**Pull request by frazjp65**
Add a way to start vim cleanly and a way to start in read-only mode. Remove information on how to use vim and add a link to vimtutor.

----

- [ ] ~~The page (if new), does not already exist in the repo.~~

- [ ] ~~The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.~~

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines

### CODE REVIEW

**Comment by frazjp65** [comment]
I removed the link to vimtutor and instead referred to it. Apparently they do not display properly from the command line.

**Comment by agnivade** [comment]
Hi @frazjp65, thanks for contributing.

I would rather keep the examples on how to use vim in this page. The prime reason being usefulness and practicality. Almost, everyone who use vim will need to use these basic search and edit commands. Otherwise, simply mentioning the `vim` command and its options is not much useful. You can see a similar thing being done for the less.md page.

However, this does not apply everywhere though. For eg. we do not mention console specific commands  for tools like `psql` or `mysql`. Those are much more tool specific and the main command has much more useful options that needs to be considered within the short limit of 8 examples that we have.

In short, its a balancing act. We try to keep a page as useful and practical possible within our limited scope.

**Comment by frazjp65** [comment]
I re-added the commands and added `vim -R`. Does it look better now?

**Comment by agnivade** [comment]
Unfortunately, we allow only 8 examples in a page. It is mentioned explicitly in the PR template.

**Comment by frazjp65** [comment]
To be fair, only 4 examples are how to use the command. However, I understand the need for the other examples.
